frontend/src/app/services/ship-generator/index-random-generator/index-random-generator.service.ts

```diff
@@ -20,19 +21,20 @@ export class IndexRandomGenerator {
     return this.generateRandomValidIndex(validBoardSize, isHorizontal);
   }
   
-  isOneIndexNotInRange(boardSize, rowIndex, columnIndex): boolean {
-    return this.isNotInRange(boardSize, rowIndex) || this.isNotInRange(boardSize, columnIndex);
-  }
-  
   private generateRandomValidIndex(boardSize: number, isHorizontal): number {
     let index = -1;
-    while (this.isNotInRange(boardSize, index)) {
+    while (IndexVerification.isNotInRangeForBoardSize(boardSize, index)) {
       index = Math.floor(Math.random() * (boardSize + 1));
     }
     return index;
   }
   
-  private isNotInRange(boardSize: number, index: number): boolean {
-    return !(index >= 0 && index < boardSize);
-  }
+  //TODO delete
+  // isOneIndexNotInRangeForBoardSize(boardSize, rowIndex, columnIndex): boolean {
+  //   return this.isNotInRangeForBoardSize(boardSize, rowIndex) || this.isNotInRangeForBoardSize(boardSize, columnIndex);
+  // }
+  //
+  // private isNotInRangeForBoardSize(boardSize: number, index: number): boolean {
+  //   return !(index >= 0 && index < boardSize);
+  // }
```

Comment:
@RafalJDev Next time when you would like to comment some code, please hit yourself in your head and remove it.